src/lib/components.ts

```diff
@@ -1,964 +1,1123 @@
-// Constants
-const sizeInt = 4;
+import config from '../config';
 
 const builtin: ComponentDefinition[] = [
     {
-        'name': 'Effect List', // builtin and r_list.cpp for extended Effect Lists
-        'code': 0xfffffffe,
-        'group': '',
-        'func': 'effectList'
+        name: 'Effect List', // builtin and r_list.cpp for extended Effect Lists
+        code: 0xfffffffe,
+        group: '',
+        func: 'effectList'
```

Comment:
Do we have to remove the quotes on the keys?

I get that it looks cleaner and is nicer to work with, but I think that if anyone (\*cough\* me \*cough\*) ever wants to make a JSON out of this again (because I consider this file in this repo to be ground truth for component fields) then it's extra work again to re-add them. I would like to keep all keys quoted.

Reply:
This is the work of the formatter. Personally, I prefer this, because it usually results in more readable syntax highlighting (unquoted keys have a different color than quotes strings). Actually, this file *could* be a JSON, but then `sizeInt` could no longer be a constant.

Otherwise I would hope that you don't convert an object to JSON by hand 😲

Reply:
Alright, I bow to the formatter. And you’re right, I should export this properly, when I need to.